src/Features/Core/Portable/EditAndContinue/EditSession.cs

```diff
@@ -55,20 +56,23 @@ internal sealed class EditSession : IDisposable
         private readonly object _documentsWithReportedDiagnosticsGuard = new();
 
         private PendingSolutionUpdate? _pendingUpdate;
-        private bool _changesApplied;
 
         internal EditSession(
             DebuggingSession debuggingSession,
             EditSessionTelemetry telemetry,
-            IManagedEditAndContinueDebuggerService debuggerService)
+            bool inBreakState)
         {
             DebuggingSession = debuggingSession;
             Telemetry = telemetry;
-            DebuggerService = debuggerService;
 
             _nonRemappableRegions = debuggingSession.NonRemappableRegions;
 
-            BaseActiveStatements = new AsyncLazy<ActiveStatementsMap>(cancellationToken => GetBaseActiveStatementsAsync(cancellationToken), cacheResult: true);
+            InBreakState = inBreakState;
+
+            BaseActiveStatements = inBreakState ?
+                new AsyncLazy<ActiveStatementsMap>(cancellationToken => GetBaseActiveStatementsAsync(cancellationToken), cacheResult: true) :
+                new AsyncLazy<ActiveStatementsMap>(ActiveStatementsMap.Empty);
```

Comment:
```suggestion
            BaseActiveStatements = inBreakState
                ? new AsyncLazy<ActiveStatementsMap>(cancellationToken => GetBaseActiveStatementsAsync(cancellationToken), cacheResult: true)
                : new AsyncLazy<ActiveStatementsMap>(ActiveStatementsMap.Empty);
```

nit.

Reply:
Looks ugly to me.

Reply:
it's the pattern we use in teh IDE.  keeps the extraneous tokens for conditionals aligned nicely.  it prevents needing to scan to the end to even realize tehre's a conditional in the first place, or which parts are which :)

Reply:
This is news for me. If it's a pattern we use it should be enforced by analyzer. I don't like it.